Parsers/ASimWebSession/Parsers/ASimWebSessionzScalerZIA.yaml

```diff
@@ -67,7 +67,9 @@ ParserQuery: |
       * "rulelabel=" RuleName:string ";"
       "ruletype=" ruletype:string ";"
       "urlclass=" urlclass:string ";"
-      "devicemodel=" * 
+      "devicemodel=;" devicemodel:string "," * 
```

Comment:
Hi @v-sudkharat , I hope this is checked with the data and this field is getting extracted correctly, thanks.